src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -337,6 +358,8 @@ private void configureApplicationManagers() {
 
         bind(IUserAlerts.class).to(PgUserAlerts.class);
 
+        bind(IUserStreaksManager.class).to(PgUserStreakManager.class);
```

Comment:
`IUserStreaksManager` has been added, instead of the singleton provider which was overkill.